server/schema/keywords.ts

```diff
@@ -41,6 +41,7 @@ enum Keyword {
   RESCRIPT= "ReScript",
   RUST = "Rust",
   SSR = "SSR",
+  SSG = "Static Site Generator",
```

Comment:
```suggestion
  SSG = "Static Site Generation",
```

Reply:
it's funny how "bad" this term is defined... I search for it and found
- Static-Site Generating
- Static-Site Generate
- Static Site Generators

I guess it's most prominent in the JAMstack and they call it `Static Site Generators`
https://jamstack.org/generators/

```suggestion
  SSG = "Static Site Generators",
```

Reply:
Yeah, there's no single answer. We can go with yours.